DeepSpace.Contracts/Ship.cs

```diff
@@ -1,12 +1,39 @@
-﻿namespace DeepSpace.Contracts
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DeepSpace.Contracts
 {
     public class Ship
     {
         public string Name { get; set; }
         public string CommandCode { get; set; }
         public string TransponderCode { get; set; }
 
+        public double BaseHealth { get; set; } = 100; // Could be based on ship type for example.
+        private double Shield => ShieldUpgrades.Sum(su => su.ShieldValue);
+        private List<IShieldUpgrades> ShieldUpgrades { get; set; }
+
+        public double CurrentHealth { get; set; }
+
         public Location Location { get; set; }
         public Move Move { get; set; }
+        public void AddShieldUpgrade(IShieldUpgrades upgrade)
```

Comment:
This class is saved directly into the database so it should just be a list of properties. Can you add these methods into ShipManager - that's where the BLL wants to live